apps/vaporgui/TFWidget.cpp

```diff
@@ -43,6 +45,13 @@ using namespace TFWidget_;
 #define CANCEL -1
 #define ACCEPT  0
 
+// std::make_unique is not available until C++14 :(
+//
+template<typename T, typename... Args>
+std::unique_ptr<T> make_unique(Args&&... args) {
```

Comment:
> Will this cause problems on a C++14 compiler? 

Nope.  make_unique was introduced in C++14.

> Also, why are you using a managed pointer for the colormap copy? Is this because it is still managed by the params database? If that is the case, this would still cause a double free.

I used smart pointers because they're recommended best-practice.  The MapperFunction copy has no parent, so its memory is not managed by anything but the unique_ptr.

Reply:
> Nope. make_unique was introduced in C++14.

Thats my point.
If you compile using C++14 you get an error:
```
TFWidget.cpp:1213:21: error: call to 'make_unique' is ambiguous
```

Reply:
I looked into it and I don't think there would be a problem with a double free here.

Reply:
Fixed

Reply:
You are still redefining the make_unique function

Reply:
My bad.  It's out now.